### PR TITLE
Polys: Improved the docstrings of LM(), LT() and revert()(#11427)

### PR DESCRIPTION
**Pull request by shrutishrm512**
Added more description about the method in LM(), LT() and gave the examples for revert(f, n) to clear about its functionality in docstrings. These changes have been made in the polytools.py file in polys.

### CODE REVIEW

**Comment by asmeurer** [comment]
There are test failures. See the Travis tests. Be sure to configure your text editor to use four spaces for tabs.

**Comment by asmeurer** [comment]
And you can run

```
./bin/doctest
```

locally before committing to make sure everything works.

**Comment by shrutishrm512** [comment]
Please can you tell me in detail about Travis tests for making changes in my code.

File: sympy/polys/polytools.py [hunk 1]
```diff
@@ -1979,6 +1978,9 @@ def coeff(f, x, n=1, right=False):
     def LM(f, order=None):
         """
         Returns the leading monomial of ``f``.
+
```
Comment (Shekharrajak) [inline sympy/polys/polytools.py]: inconsistent use of tabs and spaces in indentation. Use [PEP](https://packagecontrol.io/packages/Python%20PEP8%20Autoformat) plugin to cross check python code format.

**Comment by shrutishrm512** [comment]
@asmeurer  I have again sent the pull request and it has cleared all the checks, then it can be merged into the main code?

**Comment by shrutishrm512** [comment]
#11663

**Comment by asmeurer** [comment]
Thanks. And by the way, that's the same pull request. And there's no need to open a new pull request. Just push new changes to the same branch, and it will update the pull request (that's what has happened here).

**Comment by shrutishrm512** [comment]
OK sir, I will take care of this in furture.


 Sent with Mailtrack
<https://mailtrack.io/install?source=signature&lang=en&referral=shrutishrm512@gmail.com&idSignature=22>

*Shruti Sharma*

*Pre-final Year Student*
*B.Tech.:Computer Science and Engineering*
*The LNM Institute of Information Technology*
Email ID*:shrutishrm512@gmail.com <shrutishrm512@gmail.com>*

**Pull request merged**